testplan/common/utils/testing.py

```diff
@@ -131,6 +140,23 @@ def suppress_warnings(func):
     return context_wrapper(warnings_suppressed)
 
 
+def _render_mismatch(orig_act, act, orig_exp, exp, full_path):
```

Comment:
FFR this was moved out to prevent creating a brand new function on each recursion. The optimization was too easy to make so I made it.